decentraland/redux/reducers.js

```diff
@@ -67,8 +68,12 @@ function myAssets(state = { list: [] }, action) {
   switch (type) {
     case ADD_TO_MY_ASSETS_LIST:
       return { ...state, list: [myAsset, ...state.list] };
-    case REMOVE_MY_ASSET_FROM_LIST:
-      return { ...state, list: state.list.filter(val => val !== myAsset) };
+    case REMOVE_MY_ASSET_FROM_LIST: {
+      const { list: myAssets } = state;
+      const toRemove = myAsset;
+      const list = myAssets.filter(asset => asset !== toRemove);
```

Comment:
It's probably overkill, but we could dry this logic up with the reducer for land for sale if we wanted to. Doesn't need to happen in this PR or ever, though. Just a thought.

Reply:
Done on https://github.com/tasitlabs/tasit/pull/256. Resolving this.